src/components/plotly-modal/plotly-modal.scss

```diff
@@ -21,7 +21,7 @@
   right: 0;
   bottom: 0;
   left: 0;
-  z-index: 3;
+  z-index: 5;
```

Comment:
Given we now have a global toolbar, that toolbar should almost always be visible. 

With that in mind, I think we should keep the `z-index` at 3 and then bump over the the `left` value to `80px`, since that's the width of the global toolbar.

There's actually a SCSS variable you can use for this [here](https://github.com/quantumblacklabs/kedro-viz/blob/6206e54eb4f7380d18c95a69496ddd578c78087e/src/styles/_variables.scss#L68).